### PR TITLE
[18.0][MIG] uom_unece: Migration to 18.0

### PR DESCRIPTION
**Pull request by LucasTran380381**

### CODE REVIEW

**Review by henrybackman — APPROVED** [review]

**Comment by bosd** [comment]
Please squash the translation commits here as well :face_in_clouds:

File: uom_unece/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Product UoM UNECE",
-    "version": "17.0.1.0.0",
+    "version": "18.0.1.0.0",
```
Comment (sebalix) [inline uom_unece/__manifest__.py:7]: Technically, version bump and contributors updates should land in the migration commit.
pre-commit commit is only to align with the new pre-commit config and commited with `--no-verify`.

**Review by sebalix — COMMENTED** [review]
LG, small comment

File: uom_unece/__manifest__.py (hunk 1 above)
Comment (LucasTran380381) [reply uom_unece/__manifest__.py:7]: I updated commit

**Review by sebalix — APPROVED** [review]

**Comment by sebalix** [comment]
Thank you @LucasTran380381 :pray: 

/ocabot merge nobump

**Pull request merged**